plugin/src/main/java/com/google/errorprone/xplat/checker/AllowLegacyTime.java

```diff
@@ -0,0 +1,18 @@
+package com.google.errorprone.xplat.checker;
+
+import static java.lang.annotation.ElementType.CONSTRUCTOR;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.LOCAL_VARIABLE;
+
+import java.lang.annotation.Target;
+
+/**
+ * Annotation that indicates the explicit use of java.util.Calendar, in order to override the
+ * CalendarClassBan xplat error prone checker.
+ */
+@Target({CONSTRUCTOR, METHOD, PARAMETER, FIELD, LOCAL_VARIABLE})
+public @interface AllowLegacyTime {
+
```

Comment:
Add a `String value();` (along with a Javadoc describing what the value is for) to allow self-documenting strings to be used with the annotation, for example:

`@AllowLegacyTime("Legacy API Foo only accepts java.util.Calendar")`

See https://github.com/google/error-prone/blob/master/annotations/src/main/java/com/google/errorprone/annotations/concurrent/GuardedBy.java#L34 for an example of the `String value();` use and its Javadoc.